dbsync.py

```diff
@@ -453,29 +473,39 @@ def dbsync_init(mc, from_gpkg=True):
 
     # let's start with various environment checks to make sure
     # the environment is set up correctly before doing any work
+    print("Connecting to the database...")
+    try:
+        conn = psycopg2.connect(config.db_conn_info)
+    except psycopg2.Error as e:
+        raise DbSyncError("Unable to connect to the database: " + str(e))
+
+    base_schema_exists = _check_schema_exists(conn, config.db_schema_base)
+    modified_schema_exists = _check_schema_exists(conn, config.db_schema_modified)
+
+    try:
+        server_info = mc.project_info(config.mergin_project_name)
+    except ClientError as e:
+        raise DbSyncError("Mergin client error: " + str(e))
+
     gpkg_full_path = os.path.join(config.project_working_dir, config.mergin_sync_file)
     if not os.path.exists(config.project_working_dir):
         # download the Mergin project
         print("Download Mergin project " + config.mergin_project_name + " to " + config.project_working_dir)
-        mc.download_project(config.mergin_project_name, config.project_working_dir)
+        if modified_schema_exists and base_schema_exists:
+            _, version = _get_db_project_comment(conn, config.db_schema_base)
+            mc.download_project(config.mergin_project_name, config.project_working_dir, version)
```

Comment:
It would be good to add a print to both branches to state we are downloading e.g. v15 because that's what is in the database -or- we download the latest (because the schemas do not exist yet)